### PR TITLE
feat: add values for `priorityClassName`

### PR DESCRIPTION
**Pull request by pkalemba**
## What does your PR do?

Adds priorityClassName to pods

### For all Pull Requests

- [X] Commits are [signed off](https://github.com/airflow-helm/charts/tree/main/charts/airflow/CONTRIBUTING.md#sign-your-work)
- [X] Commits have [semantic messages](https://github.com/airflow-helm/charts/tree/main/charts/airflow/CONTRIBUTING.md#semantic-commit-messages)
- [X] Documentation [updated](https://github.com/airflow-helm/charts/tree/main/charts/airflow/CONTRIBUTING.md#documentation)
- [X] Passes [ct linting](https://github.com/airflow-helm/charts/tree/main/charts/airflow/CONTRIBUTING.md#linting)

### For releasing ONLY

- [X] Chart.yaml [version bumped](https://github.com/airflow-helm/charts/tree/main/charts/airflow/CONTRIBUTING.md#versioning)
- [ ] CHANGELOG.md updated

### CODE REVIEW

**Comment by pkalemba** [comment]
@thesuperzapper can You look at it please?

**Comment by pkalemba** [comment]
Can someone look at it ?

File: charts/airflow/values.yaml [hunk 1]
```diff
@@ -197,6 +197,9 @@ airflow:
     ## this does NOT give root permissions to Pods, only the "root" group
     fsGroup: 0
 
+  ## default priorityClassName for airflow Pods (is overridden by pod-specific values)
+  defaultPriorityClassName: ""
```
Comment (thesuperzapper) [inline charts/airflow/values.yaml:201]: To keep things consistent let's use the same formatting as the other docstrings:

```yaml
  ## default priorityClassName for airflow Pods (is overridden by pod-specific values)
  ## - docs for priorityClassName:
  ##   https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/
  ##
  defaultPriorityClassName: ""
```

File: charts/airflow/values.yaml [hunk 2]
```diff
@@ -412,6 +415,10 @@ airflow:
     ##
     securityContext: {}
 
+    ## the priorityClassName for the db-migrations Pods
+    ##
+    priorityClassName: ""
```
Comment (thesuperzapper) [inline charts/airflow/values.yaml:420]: Let's add a link for people who don't know what `priorityClassName` is:

```yaml
  ## the priorityClassName for the db-migrations Pods
  ## - docs for priorityClassName:
  ##   https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/
  ##
  priorityClassName: ""
```

__NOTE:__ we will also need to add it to all the other `priorityClassName` docstrings in this values file

**Review by thesuperzapper — CHANGES_REQUESTED** [review]
@pkalemba you are amazing! Thanks for this PR!

I have left a small number of comments that need to be fixed, but otherwise it looks good for the next release.

**Pull request closed without merging**